addons/mail/static/src/models/suggested_recipient_info.js

```diff
@@ -70,6 +70,13 @@ Model({
          * visual clue when displaying `this`.
          */
         reason: attr(),
+        /**
+         * Default values for partner creation (empty object if partner is set).
+         */
+        defaultCreateValues: attr({
```

Comment:
Shouldn't we order attributes/fields alphabetically in JS ?

Reply:
It’s the only one not ordered :-), thanks